src/SFML/Graphics/Font.cpp

```diff
@@ -348,7 +348,11 @@ const Font::Info& Font::getInfo() const
 const Glyph& Font::getGlyph(Uint32 codePoint, unsigned int characterSize, bool bold, float outlineThickness) const
 {
     // Get the page corresponding to the character size
-    GlyphTable& glyphs = m_pages[characterSize].glyphs;
+    PageTable::iterator pageIterator = m_pages.find(characterSize);
+    if (pageIterator == m_pages.end())
+        pageIterator = m_pages.insert(std::make_pair(characterSize, Page(m_isSmooth))).first;
+
+    GlyphTable& glyphs = pageIterator->second.glyphs;
```

Comment:
Since this code appears 3 times, would it make sense to have a dedicated function?
```cpp
Page& Font::loadPage(unsigned int characterSize);
```

Reply:
It looks much better with a dedicated function. Thank you.